distribution/openhabhome/configurations/openhab_default.cfg

```diff
@@ -2171,12 +2171,20 @@ tcp:refreshinterval=250
 #
 # MQTT broker port SiteWhere is listening to. 
 # sitewhere:mqttPort=1883
-
-################################ UCProjects.eu Relay Board Binding ###################################
-#
-# Port and baud rate (optional, defaults to 57600) for every device
-# ucprelayboard:board.one.port=/dev/tyUSB0
-# ucprelayboard:board.<name>.baud=
-#
-# Refresh of relay board state interval in miliseconds (optional, defaults to 60000)
-# ucprelayboard:refresh=
+
+################################ UCProjects.eu Relay Board Binding ###################################

```

Comment:
Your change should not have touched the ucprelayboard config section. Please check whether there were any whitespace changes (or line end signs or something like that)